src/components/Home.js

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getStocks } from 'redux/stock/stocksSlice';
+import styles from 'styles/Home.module.css';
+import Hero from './Hero';
+import StockItem from './StockItem';
+
+const Home = () => {
+  const { stocks, hero } = useSelector((store) => store.stock);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getStocks());
+  }, [dispatch]);
+
+  return (
+    <div className="container">
+      <Hero data={hero} />
+      <ul className={styles.items}>
+        {
+          stocks.map((stock) => <StockItem key={stock.symbol} data={stock} />)
+        }
+      </ul>
+    </div>
```

Comment:
- [ ] You have done a great job, especially on the design part, but you may have missed an important requirement for this project. We are told to create a search functionality where users would search for an item using it's name, I have attached an image for your reference.

![search](https://user-images.githubusercontent.com/53124548/226115247-e99d0c22-be03-4e10-92c1-0cd3be507292.png)

Reply:
Thank you @elisha2kyakpo1 
I didn't think that was part of the requirement as it was not stated under the project requirement, I will add it and request a new review.